### DISCUSSION TITLE
Parameter for selection (for custom commands)

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#348] [Feature Requests]
I want to create a command that transforms just the current selection.
As far as I see in the [parameter documentation](https://docs.nota.md/@parameters.html), this is not possible yet.
Example use case: Slugify the current selection so I can use it as slug for my blog article.

**Comment by astoilkov** [comment]
Other people have requested this as well. We are considering it. Maybe next time we spent time on extensibility we can add this. See a similar issue: https://github.com/notaapp/talk/issues/242.
